build-system/test-configs/forbidden-terms.js

```diff
@@ -510,7 +519,10 @@ const forbiddenTermsGlobal = {
       'src/service/url-replacements-impl.js',
     ],
   },
-  'debugger': '',
+  'debugger': {
+    message: 'remove debugger',
```

Comment:
Wanna standardize the error message as `Please remove all instance of debugger`?